### PR TITLE
add missing attributes in Keypair issue:#860

### PR DESCRIPTION
**Pull request by whaon**
add missing attributes in Keypair issue: #860

### CODE REVIEW

File: core/src/main/java/org/openstack4j/model/compute/Keypair.java [hunk 1]
```diff
@@ -35,5 +37,41 @@
...
+	 * time deleted
+	 * @return
+	 */
+	Date getDeletedAt();
+	
+	/**
+	 * 
+	 * @return id of the keypair
+	 */
+	Integer getId();
```
Comment (auhlig) [inline core/src/main/java/org/openstack4j/model/compute/Keypair.java]: Looking at http://developer.openstack.org/api-ref/compute/?expanded=create-or-import-keypair-detail#create-or-import-keypair - What's the `id` of the keypair?

Comment (whaon) [inline core/src/main/java/org/openstack4j/model/compute/Keypair.java]: consider the below response content that I get from the url you given
```
{
    "keypair": {
        "fingerprint": "44:fe:29:6e:23:14:b9:53:5b:65:82:58:1c:fe:5a:c3",
        "name": "keypair-6638abdb-c4e8-407c-ba88-c8dd7cc3c4f1",
        "public_key": "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC1HTrHCbb9NawNLSV8N6tSa8i637+EC2dA+lsdHHfQlT54t+N0nHhJPlKWDLhc579j87vp6RDFriFJ/smsTnDnf64O12z0kBaJpJPH2zXrBkZFK6q2rmxydURzX/z0yLSCP77SFJ0fdXWH2hMsAusflGyryHGX20n+mZK6mDrxVzGxEz228dwQ5G7Az5OoZDWygH2pqPvKjkifRw0jwUKf3BbkP0QvANACOk26cv16mNFpFJfI1N3OC5lUsZQtKGR01ptJoWijYKccqhkAKuo902tg/qup58J5kflNm7I61sy1mJon6SGqNUSfoQagqtBH6vd/tU1jnlwZ03uUroAL Generated-by-Nova\n",
        "user_id": "fake",
        "deleted": false,
        "created_at": "2014-05-07T12:06:13.681238",
        "updated_at": null,
        "deleted_at": null,
        "id": 1
    }
}
```
I think the id means the identity of the keypair

**Comment by whaon** [comment]
I just merge the code that resolve the issue: #859 
I try my best to make it correct but it may still have problems.Our Openstack environment is lower than 2.26 while the api is available since version 2.26(please refer the api document),so I can't make a test for my code.
Please **test** my code and if it has any problems,just contact me for free,Thanks!

File: core/src/main/java/org/openstack4j/model/compute/Keypair.java (hunk 1 above)
Comment (auhlig) [inline core/src/main/java/org/openstack4j/model/compute/Keypair.java]: I'm blind

**Comment by auhlig** [comment]
I'd rather like to ask you to extend or add some unit tests.

**Comment by auhlig** [comment]
Cool. Many thanks for the work @whaon, @tongh-qtl. Looks good from my side. Do you agree @vinodborole ?

**Comment by vinodborole** [comment]
LGTM

**Comment by auhlig** [comment]
@whaon Could you create a PR for https://github.com/ContainX/openstack4j.com and add documentation for the newly introduced methods? Many thanks in advance.

**Comment by whaon** [comment]
it's ok,but maybe I will do it a bit later

**Comment by auhlig** [comment]
Hey @whaon,
Right. It's just markdown and a little amount of html if you want to set some links.

**Pull request merged**